docs/use-cases.rst

```diff
@@ -196,6 +196,93 @@ Two parameters to be configured in this type of implementation are outlined:
+
+The :ref:`above example <discovery_server_major_scenario_setup>` presents a *single point of failure*, that is, if the
+*server* falls there is no discovery. In order to prevent this, several servers could be linked to a *client*. By doing
+this, a discovery failure only takes place if *all servers* fail, a more unlikely event.
+
+We must make sure that the each server has a unique **Prefix** and *unicast address*. In the example below we have
+chosen:
+
+.. csv-table::
+    :header: "Prefix", "UDPv4 address"
+    :widths: 20,100
+
+    75.63.2D.73.76.72.63.6C.6E.74.2D.31, "192.168.10.57:56542"
+    75.63.2D.73.76.72.63.6C.6E.74.2D.32, "192.168.10.60:56543"
+
+.. image:: ds_redundancy.png
+    :align: center
+    :width: 75%
+
+Note that several *servers* can share the same *IP address* but the port number should be different. Likewise several
+*servers* can share the same port if its *IP address* is different.
+
++--------------------------------------------------------+--------------------------------------------------------+
+| **SERVER**                                             | **CLIENT**                                             |
++--------------------------------------------------------+--------------------------------------------------------+
+| **C++**                                                | **C++**                                                |
++--------------------------------------------------------+--------------------------------------------------------+
+| .. literalinclude:: ../code/CodeTester.cpp             | .. literalinclude:: ../code/CodeTester.cpp             |
+|    :language: c++                                      |    :language: c++                                      |
+|    :start-after: //CONF_DS_REDUNDANCY_SCENARIO_SERVER  |    :start-after: //CONF_DS_REDUNDANCY_SCENARIO_CLIENT  |
+|    :end-before: //!--                                  |    :end-before: //!--                                  |
++--------------------------------------------------------+--------------------------------------------------------+
+| **XML**                                                | **XML**                                                |
++--------------------------------------------------------+--------------------------------------------------------+
+| .. literalinclude:: ../code/XMLTester.xml              | .. literalinclude:: ../code/XMLTester.xml              |
+|    :language: xml                                      |    :language: xml                                      |
+|    :start-after: <!-->CONF_DS_RDNCY_SCENARIO_SERVER<-->|    :start-after: <!-->CONF_DS_RDNCY_SCENARIO_CLIENT<-->|
+|    :end-before: <!--><-->                              |    :end-before: <!--><-->                              |
++--------------------------------------------------------+--------------------------------------------------------+
+
+.. _discovery_server_persistency_scenario_setup:
+
+UDPv4 persistency example
```

Comment:
In this example:

* Use reported speech, as in the rest of the sections
* I think the text needs to be more "story telling". The sentences do not feel connected to each other (in fact each one is a paragraph).
* A bit of context, you gotta think that the person reading this may not read anything else.
* I miss snippets, I thing we should have them in all the examples. Again, since this is a use-case, the read may not have read the previous sections.

Reply:
When I wrote it I was pondering if introduce cumbersome source and xml snippets for the partitioning because only the first case has an associated snippet. I thought those snippets will be the same thing again and again and confuse more than clarify. But maybe you're right, let's go for it!